fastify.js

```diff
@@ -224,6 +242,15 @@ function build (options) {
     }
 
     req.id = genReqId(req)
+    if (typeof context.trustProxyFn === 'function') {
```

Comment:
can you move this check outside in an external boolean? I would prefer if we didn't attach the function to the context - I do not want mulitple plugins to be able to override this.

Reply:
Actually I have no idea how to set this somewhere else outside of context.